src/backend/impress/settings.py

```diff
@@ -372,6 +372,12 @@ class Base(Configuration):
     SENTRY_DSN = values.Value(None, environ_name="SENTRY_DSN", environ_prefix=None)
 
     # Collaboration
+    COLLABORATION_API_URL = values.Value(
+        None, environ_name="COLLABORATION_API_URL", environ_prefix=None
+    )
+    COLLABORATION_SERVER_SECRET = values.Value(
+        None, environ_name="COLLABORATION_SERVER_SECRET", environ_prefix=None
+    )
```

Comment:
I would find it easier to understand if these 2 settings shared the same root:
COLLABORATION_SERVER_URL 
COLLABORATION_SERVER_SECRET

Reply:
I put `COLLABORATION_API_URL` because we have as well `COLLABORATION_WS_URL`, both of them target the server somehow, we need to dissociate the base url as well because of docker compose:
```
COLLABORATION_API_URL=http://nginx:8083/collaboration/api/
COLLABORATION_WS_URL=ws://localhost:8083/collaboration/ws/
 ```
 If you think it is better, I can change, no prob.